apps/fabric-website/index.html

```diff
@@ -101,11 +49,15 @@
       var entryPointFilename = 'fabric-sitev5';
       var appPath = '';
       var Flight = {};
+      // Suffix used on our built JS files. Starts out as .min.js, and if that fails to load,
+      // switches to .js (non-min). This helps us determine whether to load the min or non-min
+      // variants of the Monaco workers.
+      var jsSuffix = '.min.js';
 
       // Determine production or staging/dogfood
       if (location.hostname === 'developer.microsoft.com' || productionParam) {
         isProduction = true;
-      } else if (location.hostname === 'uifabric-tst.azurewebsites.net' || dogfoodParam) {
+      } else if (location.hostname === 'developer.microsoft-tst.com' || dogfoodParam) {
```

Comment:
I wouldn't switch this. `index.html` is actually used for the `uifabric-tst` environment, whereas `homepage.htm` is used for the main prod site on developer.microsoft.com.